### DISCUSSION TITLE
Capture image while stream is active/inactive.

### DISCUSSION THREAD

**Discussion by mcelhennyi** [HaishinKit/HaishinKit.swift#1095] [Idea]
I would like to be able to capture still frames while streaming or while not streaming.

**Comment by mcelhennyi** [comment]
Possible to expedite this feature @shogo4405 ?

**Reply by shogo4405** [reply]
HaishinKit is an OSS project. I'm waiting your PR! Thank you.

**Reply by mcelhennyi** [reply]
Would you be able to explain, to a novice iOS developer like myself, the right way to implement this feature? I am unfamiliar with iOS and your code base. So any suggestions or guidance is welcome.
